mixer/adapter/fluentd/fluentd.go

```diff
@@ -114,10 +165,39 @@ func (b *builder) SetLogEntryTypes(types map[string]*logentry.Type) {
 	b.types = types
 }
 
+func (h *handler) postData() {
+	var sendBuf bytes.Buffer
+	sendBuf.Grow(int(h.maxBatchBytes))
+	tick := time.NewTicker(h.pushInterval)
+	for {
+		select {
+		case <-h.stopCh:
+			tick.Stop()
+			if sendBuf.Len() > 0 {
+				h.logger.PostRawData(sendBuf.Bytes())
+				sendBuf.Reset()
+			}
+			return
+		case data := <-h.dataBuffer:
+			if sendBuf.Len()+len(data) > int(h.maxBatchBytes) && sendBuf.Len() > 0 {
```

Comment:
Pl add comment saying that `len(data) > h.maxBatch` is not possible because it is rejected earlier.

Reply:
OBE.